### PR TITLE
guide: move gdrive

### PR DESCRIPTION
**Pull request by dberenbaum**
Migrates Google Drive guide and command ref info to new guide section. No substantive changes. Part of #2866

### CODE REVIEW

**Review by shcheklein — CHANGES_REQUESTED** [review]
Thanks @dberenbaum ! Do we need to update redirects by chance?

File: content/docs/user-guide/data-management/remote-storage/google-drive.md [hunk 1]
```diff
@@ -47,6 +25,29 @@ Authentication successful.
 
 See [Authorization](#authorization) for more details.
 
+Note that GDrive remotes are not "trusted" by default. This means that the
```
[the remark this answers is not in the document]
Comment (dberenbaum) [reply content/docs/user-guide/data-management/remote-storage/google-drive.md]: Not sure where else to put it, so put it into a details section for now to hide it better.

**Comment by dberenbaum** [comment]
> Thanks @dberenbaum ! Do we need to update redirects by chance?

Good catch. Added for https://dvc.org/doc/user-guide/how-to/setup-google-drive-remote. I don't think we need any others since we still have the cmd ref pages and they link to these guides.

File: content/docs/user-guide/data-management/remote-storage/google-drive.md [hunk 2]
```diff
@@ -322,3 +322,120 @@ The remote must also be configured with the associated user **personal email**:
 $ dvc remote modify myremote gdrive_service_account_user_email \
               example_adress@some_google_domain.com
 ```
+
+## Configuration parameters
```
[the remark this answers is not in the document]
Comment (dberenbaum) [reply content/docs/user-guide/data-management/remote-storage/google-drive.md:335]: Added near the top.

**Review by shcheklein — APPROVED** [review]

**Comment by shcheklein** [comment]
There is some failure in deployment though. Need to check it first or please try to run locally.

**Comment by dberenbaum** [comment]
🤔 Deployment is working for me locally

**Comment by shcheklein** [comment]
I've fixed the deployment. Merging this, let's please rebase / merge main into the other branches after that.

**Pull request merged**